examples/python/extract-logs-write-to-cloud-logging/README.md

```diff
@@ -0,0 +1,66 @@
+## Overview
+
+A Python script that extracts Looker system/audit logs from [System Activity](https://docs.looker.com/admin-options/system-activity) and exports the Logs to Cloud Logging. This example tries to format the output logs like a [GCP Audit Log](https://cloud.google.com/logging/docs/audit/understanding-audit-logs) as best as possible. See [mapping](#gcp-audit-log-fields-to-looker-system-activity-mapping) for comparison between Looker System Activity Fields and GCP Audit Log Fields
+
+> **_NOTE:_**  The script extracts System Activity data from the last 10 minutes. You can then schedule this script to run every 10 minutes using a cron job or equivalent
+
+## Requirements
+- Looker Instance in which you have Admin or `see_system_activity` permission
+- Google Cloud Project with Cloud Logging API enabled
+- [pyenv](https://github.com/pyenv/pyenv#installation) installed
```

Comment:
Should add the python version and `gcloud` to this

Reply:
done